### PR TITLE
Resolve issue 125

### PR DESCRIPTION
**Pull request by rbessems**
Resolve issue 125

### CODE REVIEW

File: GraphicsAPI_Test/main.cpp [hunk 1]
```diff
@@ -255,7 +257,11 @@ int main() {
 
     // Swapchain
     const uint32_t swapchainCount = 3;
+#ifdef XR_TUTORIAL_USE_VULKAN    
     swapchainFormat = apiType == VULKAN ? VK_FORMAT_B8G8R8A8_UNORM : DXGI_FORMAT_B8G8R8A8_UNORM;
+#else
+    swapchainFormat = DXGI_FORMAT_B8G8R8A8_UNORM;
+#endif
```
Comment (rpavlik) [inline GraphicsAPI_Test/main.cpp:264]: Everything else looks fine but this and the above change looks unrelated? or was it just the change required to actually build?

Comment (rbessems) [reply GraphicsAPI_Test/main.cpp:264]: that was required to make build work on a machine without vulkan

**Review by rpavlik — APPROVED** [review]
Only that one thing looked unexpected. If it's not an accidental inclusion, though, that works fine.
